assets/js/utils/rtcUtils.ts

```diff
@@ -140,11 +190,18 @@ export const connectWebrtc = async (
               },
             };
           });
+
+          if ("enabled" in metadata) track.enabled = metadata.enabled;
```

Comment:
```suggestion
          track.enabled = metadata?.enabled || track.enabled
```

Reply:
Maybe it is a little faster, but for me it is not that clear. I'll leave it as it is.